### PR TITLE
[Feature] Enhance job result page

### PR DESCRIPTION
**Pull request by s7monk**
### Purpose

Complete the development of the front-end page of job result.

image

image

### CODE REVIEW

File: paimon-web-ui/package.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
     "@antv/x6": "^2.15.3",
     "@antv/x6-plugin-dnd": "^2.1.1",
     "@antv/x6-vue-shape": "^2.1.1",
+    "@vicons/fluent": "^0.12.0",
+    "@vicons/material": "^0.12.0",
+    "@vicons/tabler": "^0.12.0",
```
Comment (zhangmo8) [inline paimon-web-ui/package.json:23]: Why not use the existing icons to keep the icons consistent?

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  .active-button-light {
+    color: #7D818E;
+    background-color: #DBDCDE;
+    border-radius: 6px;
+    padding: 4px;
+  }
+
+  .active-button-dark {
+    color: #7D818E;
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.module.scss:44]: The same color can be extracted into scss variable.

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+      activeButton,
+      setActiveButton,
+      isDarkMode
+    }
+  },
+  render() {
+    return (
+      <div class={styles.container}>
+       <n-space class={styles.left} item-style="display: flex; align-items: center;">
+         <n-popover trigger="hover" placement="bottom-start" show-arrow={false} style="padding: 0"
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:49]: Remember to format the indented space of the code before submitting

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                    v-slots={{
+                      trigger: () => (
+                        <n-button
+                          text
+                          class={[this.isDarkMode ? styles['active-button-dark'] : styles['active-button-light']]}
```
Comment (zhangmo8) [inline paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:54]: There's no need to judge whether it's dark mode or not.

File: paimon-web-ui/package.json (hunk 1 above)
Comment (s7monk) [reply paimon-web-ui/package.json:23]: The icons that have been introduced in the project do not have the desired icon, so a new icon library is introduced.

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.module.scss (hunk 2 above)
Comment (s7monk) [reply paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.module.scss:44]: done.

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx (hunk 3 above)
Comment (s7monk) [reply paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx:49]: done.

File: paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx (hunk 4 above)
Comment (s7monk) [reply paimon-web-ui/src/views/playground/components/query/components/console/components/controls/index.tsx]: done.

**Review by zhangmo8 — APPROVED** [review]
LGTM

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Pull request merged**